portfolio/src/main/java/com/google/sps/data/BlogMessage.java

```diff
@@ -34,11 +34,7 @@ public BlogMessage(long id, String tag, String message,
     this.message = message;
     this.nickname = nickname;
     this.email = email;
-    if (messageReplies != null) {
-      this.messageReplies = messageReplies;   
-    } else {
-       this.messageReplies = new ArrayList<BlogMessage>();
-    }
+    this.messageReplies = messageReplies;   
```

Comment:
If I recall correctly, this is what Tayyaba added to check whether the passed-in messageReplies is null or not. Is this not needed anymore?

Reply:
Yes the null was coming from the "replies" property in datastore which was always going to be null since replies are now stored as BlogMessages. So by removing that property from datastore we avoid the null problem